docs/GettingStarted.rst

```diff
@@ -0,0 +1,86 @@
+.. raw:: html
+
+   <!-- This Source Code Form is subject to the terms of the Mozilla Public
+      - License, v. 2.0. If a copy of the MPL was not distributed with this
+      - file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+===============
+Getting Started
+===============
+Get started by installing Frost and running some pre-built tests.
+
+Installing Frost
+================
+
+Clone the repo::
+
+   git clone git@github.com:mozilla/frost.git
+
+Create a virtual environment::
+
+   cd frost
+   python3 -m venv venv  
```

Comment:
The preferred way for all these commands is to use the `Makefile`

If you do `make` in a fresh checkout, you get instructions that will lead you to running the tests.

That's a "convention" for secops repos, but not documented (or adhered to, completely)